### PR TITLE
[enocean] Add support for eltako rollershutter frm60

### PR DESCRIPTION
**Pull request by hofingerandi**
#10816 

* implementation of new `A5_3F_7F_EltakoFRM` is based on `A5_3F_7F_EltakoFSB`
* updated Readme.md
* updated dropdown in UI

Signed-off-by: hofingerandi <andreas.hofinger@gmx.net>

### CODE REVIEW

**Comment by hofingerandi** [comment]
The PR-build on jenkins was aborted after 4hours. Can't see any issues there (especially, the build of the enocean-addin is green). Maybe some timeout? Any advice would be appreciated...

**Comment by fwolter** [comment]
@ruggy83 can you take a look?

The failed build might not be related to your change.

File: bundles/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/eep/A5_3F/A5_3F_7F_EltakoFRM.java [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+import org.openhab.core.library.types.UpDownType;
+import org.openhab.core.types.Command;
+import org.openhab.core.types.State;
+import org.openhab.core.types.UnDefType;
+
+/**
+ *
+ * @author Andreas Hofinger
+ */
+public class A5_3F_7F_EltakoFRM extends _4BSMessage {
```
Comment (fruggy83) [inline bundles/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/eep/A5_3F/A5_3F_7F_EltakoFRM.java:33]: I have not found any documentation about this Eltako specifc EEP. However I guess that you have tested your implementation.

File: bundles/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/eep/EEPType.java [hunk 2]
```diff
@@ -399,6 +400,19 @@
...
             }),
 
+    EltakoFRM(RORG._4BS, 0x3f, 0x7f, false, false, "EltakoFRM", 0, A5_3F_7F_EltakoFRM.class, THING_TYPE_ROLLERSHUTTER,
+            0, new Hashtable<String, Configuration>() {
+                private static final long serialVersionUID = 1L;
+                {
+                    put(CHANNEL_ROLLERSHUTTER, new Configuration());
+                    put(CHANNEL_TEACHINCMD, new Configuration() {
+                        {
+                            put(PARAMETER_CHANNEL_TeachInMSG, "fff80d80");
```
Comment (fruggy83) [inline bundles/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/eep/EEPType.java:410]: Are you sure that this rollershutter can be teached in with this message? I have just found in the docs that this actuator sends a MSC message for teach in.

**Review by fruggy83 — COMMENTED** [review]
Implementation looks good. However I have not found any docs about this EEP. Could you check the teach in procedure?
Thanks a lot for your work 👍

**Review by fruggy83 — APPROVED** [review]
Everything ok from my side 👍

**Review by kaikreuzer — APPROVED** [review]
Awesome. Thank you both!

File: bundles/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/eep/A5_3F/A5_3F_7F_EltakoFRM.java (hunk 1 above)
Comment (hofingerandi) [reply bundles/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/eep/A5_3F/A5_3F_7F_EltakoFRM.java:33]: I have attached the telegram documentation from eltako to the related issue: #10852

File: bundles/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/eep/EEPType.java (hunk 2 above)
Comment (hofingerandi) [reply bundles/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/eep/EEPType.java:410]: I am just ignoring the MSC-telegramm; the teach-in telegram `"fff80d80"` is independent of the data the actuator sent beforehand.
I suppose, correct handling of the MSC-telegramm could enable auto-discovery in openHAB? Currently, when I try auto-discovery and start the teach-in, it just displays the `FRM` as a push-button. I then use the displayed ID, to create the `Thing` manually.

**Pull request merged**